frontend/src/config/types.ts

```diff
@@ -596,6 +596,9 @@ export interface MenuItemType {
 export type DateItem = {
   displayDate: number; // Date that will be rendered in the calendar.
   queryDate: number; // Date that will be used in the WMS request.
+  isStartDate?: boolean;
+  isEndDate?: boolean;
+  validity?: Validity;
```

Comment:
Great work @HarryMytilinaios 

Quick q - do we actually need `validity`? knowing isStartDate or isEndDate should be enough info, no?
cc @JulienPathtech

Reply:
I don't think so,
I will remove it after I fixe the `TimelineItems` if there is no usage.

Reply:
@ericboucher no we don't actually need it on a second thought, I just added it there just in case. For sure we don't need the serverAvailable dates inside the component so it does not need to take something from redux right now every information is inside the selectedLayers we pass as a prop with the custom property dateItems

Reply:
I remove `validity` from `DateItem`